BUILD.bazel

```diff
@@ -75,3 +75,9 @@ go_library(
         "//vendor/github.com/GoogleCloudPlatform/runtimes-common/ctc_lib:go_default_library",
     ],
 )
+
+go_binary(
```

Comment:
what's this rule for? I removed it in a previous PR because I didn't think it was being used by anything. our release automation builds `container_structure_test` instead

Reply:
weird, alright wasn't sure if you added it.